datasets/gov/vsdfv/sdi.csv

```diff
@@ -0,0 +1,16 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vsdfv/sdi,,,,,,,,,,,,,
+,,sodraimokos,,,,csv,,https://atvira.sodra.lt/downloads/lt-eur/apdraustieji_ketv.zip,"self.extract(""zip"")[""apdraustieji_ketv.csv""]",,,,,
+,,,,Imokos,,,"apdr_kateg_grupe, apdr_kateg_rusis",,,,,,,
+,,,,,metai,date,Y,METAI,,2,open,,metai,Metai
```

Comment:
Brandos lygis turėtu būti 4.

Reply:
Pačiuose duomenyse Metai pateikti nestandartiškai, pvz. 2009.0, o ne 2009, dėl to brandos lygis buvo pasirinktas 2